EIPS/eip-1620.md

```diff
@@ -1,296 +1,185 @@
-6. If the stream period ended and it was not previously stopped by any party, the payee is entitled to withdraw all the deposited funds.
+The following describes a standard whereby time is measured using block timestamps and streams are mappings in a master contract.
+
+1. Developer sets up a money streaming contract.
+2. A payer, henceforth referred to as sender, can create a stream by depositing funds in the contract and choosing a streaming duration.
+3. A payee, henceforth referred to as recipient, can withdraw money from the stream based on its continuous solvency. That is, `payment * (current block timestamp - starting block timestamp)`.
+4. The stream can be terminated by either the sender or the recipient.
+5. If the stream ended and has not been terminated, the recipient is entitled to withdraw all the remaining balance.
 
 ## Motivation
-This standardised interface aims to change the way we think about long-term financial commitments. Thanks to blockchains, payments need not be sent in chunks (e.g. monthly salaries), as there is much less overhead in paying-as-you-go. Money as a function of time would better align incentives in a host of scenarios.
+
+This standardised interface takes a stab at changing the way we think about trust in financial contracts. Thanks to blockchains, payments need not be sent in lumps (as with monthly salaries), as there is less overhead in pay-as-you-go. Money as a function of time would better align incentives in a host of scenarios.
 
 ### Use Cases
 
-This is just a preliminary list of use cases. There are other spooky ideas interesting to explore, such as time-dependent disincetivisation, but, for brevity, we have not included them here.
+This is just a preliminary list of use cases. There are other interesting ideas left to explore, such as time-dependent disincetivisation, but, for brevity, we have not included them here.
 
 - Salaries
+- Insurance
 - Subscriptions
 - Consultancies
-- CDPs
 - Rent
 - Parking
+- Lending
 
-### Crowdsales
-[RICOs](https://github.com/lukso-network/rico), or Reversible ICOs, were introduced at Devcon4 by @frozeman. The idea is to endow investors with more power and safety guarantees by allowing them to "reverse" the investment based on the evolution of the project. We previously discussed a similar concept called SICOs, or Streamable ICOs, in this research [thread](https://ethresear.ch/t/chronos-a-quirky-application-proposal-for-plasma/2928/14?u=paulrberg).
-
-Instead of investing a lump sum and giving the money away to the project developers, funds are held in a smart contract which allocates money based on the passage of time. Project developers can withdraw funds as the stream stays active, while investors have the power to get back a significant percentage of their initial commitment if the project halts.
+[Sablier](https://sablier.finance) is a champion project for this ERC. It is what we call the protocol for real-time finance.
```

Comment:
I'm not sure this paragraph is relevant here, but will defer to @MicahZoltu's judgement.